python/lsst/ctrl/oods/fileIngester.py

```diff
@@ -227,28 +225,12 @@ async def ingest(self, msg):
         # Success or failure is noted in a message description which
         # is sent via RabbitMQ message back to Archiver, which will
         # send it out via a CSC logevent.
-        code = self.SUCCESS
-        description = None
-        for butler in self.butlers:
-            (status_code, status_msg) = self.ingest_file(butler, msg)
-            if status_code == self.FAILURE:
-                code = self.FAILURE
-            if description is None:
-                description = status_msg
-            else:
-                description = f"{description}; {status_msg}"
-
-        if code == self.SUCCESS and description is None:
-            LOGGER.info("Error in processing, no success message was created")
-            return
-
-        if self.base_broker_url is not None:
-            LOGGER.info(f"Sending message: {msg}")
-            d = dict(msg)
-            d['MSG_TYPE'] = 'IMAGE_IN_OODS'
-            d['STATUS_CODE'] = code
-            d['DESCRIPTION'] = description
-            await self.publisher.publish_message(self.PUBLISH_QUEUE, d)
+        try:
+            for butler in self.butlers:
+                self.ingest_file(butler, msg)
+        except Exception as e:
+            print("Exception thrown")
+            print(f"Exception: {e}")
```

Comment:
Unless I'm missing something, the comment preceding this change needs to be either removed or updated as it no longer accurately describes what the code does.

Reply:
And are we sure we don't want a `LOGGER.exception("something")` here instead of a print?

Reply:
Comment fixed, and @timj yes, you're of course correct.  Fixed.